src/components/molecules/ReviewForm/index.jsx

```diff
@@ -0,0 +1,279 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React, { Component } from "react";
+// Router
+import { withRouter } from "react-router-dom";
+
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import { MDBIcon, MDBBtn, MDBRating } from "mdbreact";
+
+//> Redux
+// Connect
+import { connect } from "react-redux";
+// Actions
+import {
+  createReview,
+  getReviews,
+  editReview,
+} from "../../../store/actions/reviewActions";
+import ContentLoader from "react-content-loader";
+//#endregion
+
+//#region > Data
+const tooltips = [
+  {
+    tooltip: "Sehr schlecht",
+  },
+  {
+    tooltip: "Schlecht",
+  },
+  {
+    tooltip: "Akzeptabel",
+  },
+  {
+    tooltip: "Gut",
+  },
+  {
+    tooltip: "Ausgezeichnet",
+    choosed: true,
+  },
+];
+//#endregion
+
+//#region > Compoments
+class ReviewForm extends Component {
+  state = { title: "", message: "", rating: 5 };
+
+  componentDidMount() {
+    this.props.getReviews();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (
+      this.props.reviewData.data &&
+      this.props.reviewData.data !== prevProps.reviewData.data
+    ) {
+      this.getExistingReview();
+      this.setState({ searchedExisting: true });
+    }
+  }
+
+  changeHandler = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  handleRatingChange = (value) => {
+    this.setState((prevState) => {
+      if (prevState.rating !== value.value) {
+        return { rating: value.value };
+      }
+    });
+  };
+
+  handleEditRatingChange = (value) => {
+    this.setState((prevState) => {
+      if (prevState.editRating !== value.value) {
+        return { editRating: value.value };
+      }
+    });
+  };
+
+  createReview = (event) => {
+    event.preventDefault();
+    const newReview = {
+      title: this.state.title,
+      message: this.state.message,
+      rating: this.state.rating,
+      timestamp: Date.now(),
+      username:
+        this.props.profile.firstname +
+        " " +
+        this.props.profile.lastname.split("")[0] +
+        ".",
+      product: btoa("gid://shopify/Product/" + this.props.item.product_id),
+      checkout: this.props.itemCheckout,
+    };
+
+    this.props.createReview(newReview);
+    this.props.getReviews();
+  };
+
+  editReview = (event) => {
+    event.preventDefault();
+
+    const newReview = {
+      title: this.state.editTitle,
+      message: this.state.editMessage,
+      rating: this.state.editRating,
+    };
+
+    this.props.editReview(this.state.existingReview, newReview);
+  };
```

Comment:
```suggestion
  editReview = (event) => {
    const newReview = {
      title: this.state.editTitle,
      message: this.state.editMessage,
      rating: this.state.editRating,
    };
    
    this.props.editReview(this.state.existingReview, newReview);
    event.preventDefault();
  };
```

When this is possible and does not break the code.. implement it like that xd